lib/git-pr-release.js

```diff
@@ -1,14 +1,24 @@
-const detect = require('./detect');
-const Octokit = require('@octokit/rest');
-const core = require('@actions/core');
-const messageUtils = require('./message');
+const core = require("@actions/core");
+const detect = require("./detect");
+const octokitWithThrottling = require("./octokitWithThrottling");
+const messageUtils = require("./message");
+
+module.exports = async function ({
+  host,
+  token,
+  owner,
+  repo,
+  base,
+  head,
+  assign,
+  labels,
+  template,
+}) {
+  const octokit = octokitWithThrottling({
```

Comment:
octokit instance create here and passed to detect function which mean that detect will never need to create its own octokit instance